giddy-banana/models/articletopeditionsModel.cjs

```diff
@@ -0,0 +1,31 @@
+module.exports = (db) => {
+  db.Article.belongsToMany(db.TopEditions, { through: "ArticleTopEditions" });
+  db.TopEditions.belongsToMany(db.Article, { through: "ArticleTopEditions" });
+};
```

Comment:
Move this to the static method associate of the model below. The associate method is called automatically by models/index.cjs when setting up the models, so we don't have to specify it here